orderer/common/multichannel/registrar.go

```diff
@@ -800,3 +840,11 @@ func (r *Registrar) loadJoinBlocks() map[string]*cb.Block {
 
 	return channelToBlockMap
 }
+
+func (r *Registrar) removeJoinBlock(channelID string) error {
+	if err := r.joinBlockFileRepo.Remove(channelID); err != nil {
+		return errors.Wrapf(err, "failed removing joinblock for channel %s", channelID)
```

Comment:
Why are we wrapping an error we generate? Why not simply 'WithMessage', looking up the tree, it looks like these errors are already generated with a stack.

Reply:
I guess I've just been a little lazy with differentiating between Wrap and WithMessage lately... plus it's what the rest of the file (and maybe package?) does with the errors. I think it would be worth a single pass through to go with the "standard" we set forth for fabric: Wrap an error that comes from an external library and use WithMessage for the errors that fabric already generated using pkg/errors.